node/src/components/chain_synchronizer/operations.rs

```diff
@@ -1866,11 +2281,7 @@ async fn execute_blocks(
 
         while !blocks_match {
             // Could be wrong approvals - fetch new sets of approvals from a single peer and retry.
-            for peer in ctx
-                .get_filtered_fully_connected_non_joiner_peers()
-                .await
-                .into_iter()
-            {
+            for peer in get_filtered_fully_connected_peers(ctx).await.into_iter() {
```

Comment:
`into_iter()` is implicit in `for` loops.

```suggestion
            for peer in get_filtered_fully_connected_peers(ctx).await {
```

Reply:
Shouldn't Clippy have complained about that? :thinking:

Reply:
Fixed in [9219bb7](https://github.com/casper-network/casper-node/pull/3134/commits/9219bb78b19adb5d0b7efe00bfcca90432c67767).